modules/apps/27-interchain-accounts/types/account_test.go

```diff
@@ -52,6 +52,27 @@ func (suite *TypesTestSuite) TestParseAddressFromVersion() {
 
 	addr := types.ParseAddressFromVersion(version)
 	suite.Require().Equal(TestOwnerAddress, addr)
+
+	addr = types.ParseAddressFromVersion("test-version-string")
+	suite.Require().Empty(addr)
+}
+
+func (suite *TypesTestSuite) TestParseCtrlConnSequence() {
+	portID, err := types.GeneratePortID(TestOwnerAddress, "connection-0", "connection-1")
+	suite.Require().NoError(err)
+
+	connSeq := types.ParseControllerConnSequence(portID)
+	suite.Require().Equal("0", connSeq)
+	suite.Require().Empty(types.ParseControllerConnSequence(types.PortID))
```

Comment:
Should this have failed test cases as well?

Reply:
I'll update these when I change the function sig to `(uint64, error)`!